DSCResources/CommonResourceHelper.psm1

```diff
@@ -0,0 +1,142 @@
+﻿<#
+    .SYNOPSIS
+    Tests if the current machine is a Nano server.
+#>
+function Test-IsNanoServer
+{
+    [OutputType([Boolean])]
+    [CmdletBinding()]
+    param ()
+
+    return $PSVersionTable.PSEdition -ieq 'Core'
```

Comment:
Thanks @TravisEz13 - I can't actually track down the code you're referring to - I thought it was in xTimezone, but I can't find it anywhere. Do you recall where it was?

However, I actually copied this code straight out of the new [PSDscResources](https://github.com/PowerShell/PSDscResources/blob/dev/DscResources/CommonResourceHelper.psm1#L5) module, so if this code is wrong, we should change it in PSDscResources as well (it is used in [xPSDesiredStateConfiguration](https://github.com/PowerShell/xPSDesiredStateConfiguration/blob/dev/DSCResources/CommonResourceHelper.psm1#L5) too).